src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -112,8 +119,11 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
       issueReply using updatedData
   }
 
+  var isDone = false
+
   when (Done) {
     case Event(IssueReply, data) =>
+      isDone = true
```

Comment:
I've checked with my team and it looks like we want this working in server mode too which the existing PR doesn't do. I'll look into that, add a conf flag, and then push up another version.